geofield.js

```diff
@@ -268,3 +268,10 @@ function geofield_map_container_id(entity_type, entity_id, field_name, delta) {
   catch (error) { console.log('geofield_map_container_id - ' + error); }
 }
 
+/**
+ * Implements hook_locale().
+ */
+function geofield_locale() {
+  // Tell DrupalGap we have custom Spanish and Italian translations to load.
+  return ['it'];
```

Comment:
@kinmenRP This should probably be `return['it', 'zh-hans', 'zh-hant']` so DrupalGap knows to load each language file.